### PR TITLE
Added Mobile Menu

### PR DESCRIPTION
**Pull request by Abmart0803**
Added mobile menu branch.

### CODE REVIEW

**Review by addod19 — CHANGES_REQUESTED** [review]
### Hi @Abmart0803 and @AbbasSarwar, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- [ ] Github flow followed correctly :+1: 
- [ ] README file is descriptive and has all the main headers :100: 
- [ ] HTML/CSS linters set up correctly :1st_place_medal: 
- [ ] Hamburger functionality works as expected :rocket: 
- [ ] Figma design has been met :+1: 


## Required Changes ♻️
_Check the comments under the review._
- [ ] It would be better if we add linters to check our JS files. Here is a [link](https://github.com/microverseinc/linters-config/tree/master/html-css-js) on how to set up eslint

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Capitalizing all commit messages :man_technologist: 
- Committing equally..... Sharing tasks

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css [hunk 1]
```diff
@@ -1,670 +1,702 @@
...
-  height: 3px;
-  margin: 2px auto;
-  -webkit-transition: all 0.3s ease-in-out;
-  transition: all 0.3s ease-in-out;
-  background-color: #162ae3;
+.menu-close-btn {
+  position: fixed;
+  border: 2px solid #67798e;
+  top: 20px;
+  right: 30px;
```
Comment (leonardodiasb) [inline style.css:25]: - [ ] Your mobile menu works fine, but the position of the close button is a little bit off the screen. It is not possible for the user to see the close button when the menu is open. Kindly position the close button on the top right corner of the menu.
 - Your design:
 - 
image

**Review by leonardodiasb — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ⚠️ 🚦 

Hi @Abmart0803, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

mario_ladders

## Good points 👍 

 - No linter errors. ✔️
 - Correct use of GitHub flow. ✔️ 

## Aspects to improve ♻️

- [x] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by PinkMoon25 — APPROVED** [review]
Hi @Abmart0803 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
linter checks :heavy_check_mark: 
github flow :heavy_check_mark: 
Readme :heavy_check_mark: 
Mobile Menu :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**